2_large_language_models/llm_masterclass/__init__.py

```diff
@@ -0,0 +1 @@
+hello_world = "hello world"
```

Comment:
Is this needed?

Reply:
I have it in here as an example of how to create a Python package, and then install it in editable mode. In the tests, I show how you'd import this hello world, and assert it has the correct value.